operators/operator_sdk/ansible/osdk-ansible-tutorial.adoc

```diff
@@ -46,3 +47,4 @@ include::modules/osdk-create-cr.adoc[leveloffset=+1]
 == Additional resources
 
 - See xref:../../../operators/operator_sdk/ansible/osdk-ansible-project-layout.adoc#osdk-ansible-project-layout[Project layout for Ansible-based Operators] to learn about the directory structures created by the Operator SDK.
+- If a cluster-wide egress proxy is configured, cluster administrators can see xref:../../../operators/admin/olm-configuring-proxy-support.adoc#olm-configuring-proxy-support[Configuring proxy support in Operator Lifecycle Manager] for steps on overriding the cluster settings or injecting a custom CA certificate. Operator developers can use this environment to test Operator proxy support.
```

Comment:
This sounds slightly awkward with "admins can see <link>". Maybe consider making it an inline link? like:

```
If a cluster-wide egress proxy is configured, cluster administrators can xref:../../../operators/admin/olm-configuring-proxy-support.adoc#olm-configuring-proxy-support[override the proxy settings or inject a custom CA certificate].
```

or whatever the appropriate text would be